cfme/tests/intelligence/test_reports_with_timelines.py

```diff
@@ -125,7 +125,7 @@ def generate_policy_event(request, appliance, provider, vm_crud, register_event)
     request.addfinalizer(policy.delete)
 
     policy.assign_events("VM Create Complete")
-    request.addfinalizer(policy.assign_events)
+
```

Comment:
do you think unassign isn't necessary here ?

Reply:
In this case, no assign is necessary, since the policy is deleted at the end of the test anyway. You can delete a policy without unassigning events.